config/default/kustomization.yaml

```diff
@@ -23,16 +23,7 @@ resources:
 - namespace.yaml
 
 patchesStrategicMerge:
-  # Protect the /metrics endpoint by putting it behind auth.
-  # Only one of manager_auth_proxy_patch.yaml and
-  # manager_prometheus_metrics_patch.yaml should be enabled.
-- manager_auth_proxy_patch.yaml
-  # If you want your controller-manager to expose the /metrics
-  # endpoint w/o any authn/z, uncomment the following line and
-  # comment manager_auth_proxy_patch.yaml.
```

Comment:
I'd be inclined to do what they recommend and comment this out, in case we break the kubebuilder manifest generation by removing it.

Reply:
What happens when we upgrade to a newer version of Kubebuilder, won't it restore all the deleted Kustomize patches.

Reply:
Kubebuilder does not do upgrades currently, so I wouldn't be *too* concerned about this 😄